libsolidity/codegen/ArrayUtils.cpp

```diff
@@ -455,10 +444,10 @@ void ArrayUtils::copyArrayToMemory(ArrayType const& _sourceType, bool _padToWord
 			m_context.appendJumpTo(loopEnd);
 			m_context << longByteArray;
 		}
-		// compute memory end offset
-		if (baseSize > 1)
+		else
```

Comment:
Isn't this logic a bit different? Before it checked for `baseSize > 1` which probably is equal to `isByteArray()`, but now it also needs to have` _padToWordBoundaries == true` to be evaluated.

Reply:
ooh nevermind, the hidden code in github made it look like it belonged to a different `if()`.